src/Domain/FaceNormal.hpp

```diff
@@ -86,6 +101,32 @@ struct UnnormalizedFaceNormalCompute
   using volume_tags = tmpl::list<ElementMap<VolumeDim, Frame>>;
 };
 
+template <size_t VolumeDim>
+struct UnnormalizedFaceNormalMovingMeshCompute
+    : db::ComputeTag,
+      UnnormalizedFaceNormal<VolumeDim, Frame::Inertial> {
+  using base = UnnormalizedFaceNormal<VolumeDim, Frame::Inertial>;
+  static constexpr tnsr::i<DataVector, VolumeDim, Frame::Inertial> (*function)(
+      const ::Mesh<VolumeDim - 1>&, const ::ElementMap<VolumeDim, Frame::Grid>&,
+      const domain::CoordinateMapBase<Frame::Grid, Frame::Inertial, VolumeDim>&,
+      double,
+      const std::unordered_map<
+          std::string,
+          std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&,
+      const ::Direction<VolumeDim>&) = unnormalized_face_normal;
+  using argument_tags =
+      tmpl::list<Mesh<VolumeDim - 1>, ElementMap<VolumeDim, Frame::Grid>,
+                 CoordinateMaps::Tags::CoordinateMap<VolumeDim, Frame::Grid,
+                                                     Frame::Inertial>,
+                 ::Tags::Time, domain::Tags::FunctionsOfTime,
```

Comment:
you omit other `domain::Tags` namespaces, so also do so for `FunctionsOfTime` (another occurrence below)